tests/test_processing_input.py

```diff
@@ -188,6 +188,38 @@ def test_get_file_paths():
     assert len(all_files) == 4
 
 
+def test_get_file_paths_ultra():
+    # Example where we have 2 ultra 45 sensor files and 1 ultra 90 sensor file
+    # Also have an unrelated mag file.
+    # Test that we can get all the ultra 45/90 files, and also filter by sensor
+    ultra_sci_45sensor = ScienceInput(
+        "imap_ultra_l1c_sci-45sensor-pset_20240312_v000.cdf",
+        "imap_ultra_l1c_sci-45sensor-pset_20240313_v000.cdf",
```

Comment:
btw, for ENA, descriptor should start with `sensor45` if that sensor information contains in the descriptor.
```suggestion
        "imap_ultra_l1c_45sensor-pset_20240312_v000.cdf",
        "imap_ultra_l1c_45sensor-pset_20240313_v000.cdf",
```